google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateClusterRequest.java

```diff
@@ -104,6 +105,19 @@ public CreateClusterRequest setStorageType(@Nonnull StorageType storageType) {
     return this;
   }
 
+  /**
+   * Sets the Google Cloud Key Management Service (KMS) key for a CMEK-protected Bigtable.
+   *
```

Comment:
same as above, please specify that this needs the full resource path

Reply:
Done.